docs/mkdocs/changelog.md

```
@@ -0,0 +1,5 @@
+---
+hide:
+  - navigation
+---
+# Changelog
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)